mupen64plus-video-angrylion-plus/upstream/src/output/screen.h

```diff
@@ -9,4 +9,5 @@ void screen_init(struct n64video_config* config);
 void screen_adjust(int32_t width_out, int32_t height_out, int32_t* width, int32_t* height, int32_t* x, int32_t* y);
 void screen_update(void);
 void screen_toggle_fullscreen(void);
+void screen_resolution_reset(void);
```

Comment:
This is going to be automatically removed next time I pull the latest angrylion-rdp-plus. I would recommend making a pull request here with proper justification:

https://github.com/ata4/angrylion-rdp-plus